contracts/StakedCelo.sol

```diff
@@ -171,4 +180,20 @@ contract StakedCelo is ERC20Upgradeable, UUPSOwnableUpgradeable, Managed {
         }
         IManager(manager).transfer(from, to, amount);
     }
+
+    function _approve(
```

Comment:
Im wondering whether we should introduce higher granularity of Pausing since disallowing to transfer tokens seems to be rather intrusive.

Reply:
Note that the only things that get paused in this contract are ERC20 transfers, approves, and `unlockVoteBalance`. The latter can be separately paused by pausing Manager (since it calls the pausable `Manager.updateHistoryAndReturnLockedStCeloInVoting`), so there is a way to keep ERC20 functionality going when we're certain that it's a different part of the system that's the problem. 

As mentioned [in my other comment](https://github.com/celo-org/staked-celo/pull/164#discussion_r1342851647), I think it makes sense to keep the granularity simple, both to keep the implementation simple, and to reduce incident response time. Better to pause too much quickly, than spend extra time on deciding what exactly should be paused.

What do you think?